test/nodes/test_prompt_node.py

```diff
@@ -34,7 +34,7 @@ def supports(cls, model_name_or_path: str, **kwargs) -> bool:
 
 @pytest.fixture
 def get_api_key(request):
-    if request.param == "openai":
+    if request.param == "openai" or request.param == "chatgpt":
```

Comment:
I know this is tempting; I would have done the same. But we just found that chatgpt is going to be available on azure as well. Therefore, we should add a second parameter - the model name. The first parameter is the infra [hf|openai|azure], and the second is model [any_hf_model|davinci|chatgpt]. But don't worry, you don't need to do this. We'll do that in a separate PR. For now, please remove all chatgpt parametrization. And let's keep only `test_chatgpt_direct_prompting` and `test_chatgpt_direct_prompting_w_messages `

Reply:
Ok. I'll do what you say.

In any case, I'm happy that this new invocation layer already passes several tests... 🙂